ClimateMonitorV2/views/index.pug

```diff
@@ -21,14 +21,17 @@ block content
                 a(href="https://github.com/dambem/ClimateMonitorV2", id="sheffsenseImageHolder", target="_blank")
                     img(class="img-fluid", id="sheffsenseImage", src="sheffsense.png", alt="The Sheffsense logo: A colourful mosaic circle on a blurred landscape background")
             div(class="sidebar-section-active")
-                img(class="sidebar-icon" src="https://img.icons8.com/ios/20/000000/home.png")
+                img(class="sidebar-icon" src="icons/home.svg")
                 a(class="nav-link" href="/") Home
             div(class="sidebar-section")
-                img(class="sidebar-icon" src="https://img.icons8.com/ios/20/000000/about.png")
+                img(class="sidebar-icon" src="icons/info.svg")
                 a(class="nav-link" href="/about") About
             div(class="sidebar-section")
-                img(class="sidebar-icon" src="https://img.icons8.com/ios/20/000000/statistics.png")
+                img(class="sidebar-icon" src="icons/statistics.svg")
                 a(class="nav-link" href="/detailedstats") Statistics
+            div(class="sidebar-section")
+                img(class="sidebar-icon" src="icons/group.svg")
+                a(class="nav-link" href="/our-team") Our Team
             div(class="sidebar-footer")
                 a(href="https://urbanflows.ac.uk/" target="_blank")
```

Comment:
Urban flows logo is really far down in the sidebar footer on all pages again. Could we add some css to push it up a bit?